wowchemy/test/go.mod

```diff
@@ -2,6 +2,6 @@ module github.com/wowchemy/wowchemy-hugo-modules/test
 
 go 1.15
 
-require github.com/wowchemy/wowchemy-hugo-modules/wowchemy master
+require github.com/wowchemy/wowchemy-hugo-modules/wowchemy v0.0.0-20210105225557-fe24b613d566
```

Comment:
@malmarz you have pushed changes to `test/go.mod` again - please revert this. You can change the file locally if you want without committing changes to it and pushing it to the PR.

Reply:
fixed